client/login/controller.js

```diff
@@ -139,3 +139,13 @@ export function redirectDefaultLocale( context, next ) {
 		context.redirect( '/log-in' );
 	}
 }
+
+export function redirectJetpack( context, next ) {
```

Comment:
This is something that, ostensibly, should be handled elsewhere. Let's document the motivations and requirements for removal so that it's not inadvertently _ripped out_ in a cleanup effort.